serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -729,14 +738,57 @@ export class _Exporter {
                 }
                 break;
             }
+            case VertexBuffer.MatricesIndicesKind:
+            case VertexBuffer.MatricesIndicesExtraKind: {
+                for (let k = 0, length = meshAttributeArray.length / stride; k < length; ++k) {
+                    index = k * stride;
+                    const vertexData = Vector4.FromArray(meshAttributeArray, index);
+                    vertexAttributes.push(vertexData.asArray());
+                }
+                break;
+            }
+            case VertexBuffer.MatricesWeightsKind:
+            case VertexBuffer.MatricesWeightsExtraKind: {

```

Comment:
Upon further thought, I realize that this structure won't quite work for export of matrix weights, since we're dependent on both the MatricesWeights and MatricesWeightsExtra vertex attributes inorder to make sure that the joint weightings sum to 1. I think I'll have to handle joint weights as a special case, outside of the current attribute export structure...